zorg/buildbot/builders/ClangBuilder.py

```diff
@@ -278,7 +278,8 @@ def _getClangCMakeBuildFactory(
     f = LLVMBuildFactory(
             depends_on_projects=depends_on_projects,
             llvm_srcdir='llvm',
-            enable_runtimes=enable_runtimes)
+            enable_runtimes=enable_runtimes,
+            clean=clean)
```

Comment:
What is the purpose of `clean` here? It seems LLVMBuildFactory (and base BuildFactory) does not use `clean`. I see only `TODO: Handle clean property and self.clean attribute.` in zorg/buildbot/process/factory.py, line 175.
I don't see any usage of `f.clean` in ClangBuilder.py too.